warehouse/organizations/tasks.py

```diff
@@ -42,6 +42,20 @@ def update_organization_invitation_status(request):
         try:
             token_service.loads(invite.token)
         except TokenExpired:
+            invite.user.record_event(
+                tag=EventTag.Account.OrganizationRoleExpireInvite,
+                ip_address=request.remote_addr,
+                additional={
+                    "organization_name": invite.organization.name,
+                },
+            )
+            invite.organization.record_event(
+                tag=EventTag.Organization.OrganizationRoleExpireInvite,
+                ip_address=request.remote_addr,
+                additional={
+                    "target_user_id": str(invite.user.id),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏would it make sense to also record the invited user's username? I know we recently re-added searching by user id, but it may help to see the name in the Organization events.

Reply:
The idea is to someday allow username changes, so this is pre-work to ensure that at least Organization event logs will be reliable if/when that happens. Though I suppose username at time of invite/revoke/expire is worth considering.

For display I imagined we address "rehydrating" the object ids to a string by some sort of helper for events.

I think it's a larger topic worth tracking in an issue.